jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleHelpfulSuggestions.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.gradle;
+
+import com.google.cloud.tools.jib.plugins.common.HelpfulSuggestions;
+
+/** Builder for Gradle-specific {@link HelpfulSuggestions}. */
+class GradleHelpfulSuggestions extends HelpfulSuggestions {
+
+  GradleHelpfulSuggestions(String messagePrefix) {
+    super(messagePrefix, "gradle clean", "jib.to.image", "--image", "build.gradle");
```

Comment:
note: this class is renamed from `GradleHelpfulSuggestionsBuilder` below after simplified.